Project.toml

```diff
@@ -29,6 +31,7 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 WeightedOnlineStats = "bbac0a1f-7c9d-5672-960b-c6ca726e5d5d"
 YAXArrayBase = "90b8fcef-0c2d-428d-9c56-5f86629e9d14"
+Zarr = "0a941bbe-ad1d-11e8-39d9-ab76183a1d99"
```

Comment:
This would add Zarr and NetCDF as direct dependencies to the package. @felixcremer we should discuss how to deal with this. Maybe we could add them as weakdeps and only do the precompilation if these packages are in the environment?